### PR TITLE
Add 50 MHz harness to harness list.

### PR DESCRIPTION
**Pull request by litghost**

### CODE REVIEW

File: fuzzers/Makefile [hunk 1]
```diff
@@ -90,6 +90,7 @@ $(eval $(call fuzzer,035a-iob-idelay,005-tilegrid))
 $(eval $(call fuzzer,036-iob-ologic,005-tilegrid))
 $(eval $(call fuzzer,037-iob-pips,005-tilegrid))
 $(eval $(call fuzzer,038-cfg,005-tilegrid))
+$(eval $(call fuzzer,039-hclk-config,005-tilegrid))
```
Comment (mithro) [inline fuzzers/Makefile]: Should this be in a different pull request?

Comment (litghost) [reply fuzzers/Makefile]: It is (#1014), but the CI for this PR will be red if it isn't included.
